tests/test_vdx_helper.py

```diff
@@ -392,15 +383,15 @@ def test_verify_by_certificate(self, header, requests):
                                                                  file_content_type=file_content_type,
                                                                  mapper=json_mapper)
         self.assertEqual(f"{self.api_url}/verify/upload/certificate", requests.post.call_args[0][0])
-        self.assertDictEqual(verification_response, verification_response_json)
+        self.assertDictEqual(verification_response, verification_json)
```

Comment:
shouldn't it be verification_response_json?

Reply:
I renamed these examples a little bit. the "response_json" ones contain the full "response" structure (including "meta" field), and are to be used only to mock the server response. The "non response" ones contain the expected result after passing through our mappers. The json mapper cuts out the meta field, so yeah, this is the correct comparison here.

Reply:
ugh, totally missed the mapper part! thank you!

Reply:
No worries!